sys/linux/vnet.txt

```diff
@@ -1274,6 +1274,72 @@ icmpv6_ni_packet {
 	data	array[int8]
 } [packed]
 
+
+icmpv6_ndisc_option_types = ND_OPT_SOURCE_LL_ADDR, ND_OPT_TARGET_LL_ADDR, ND_OPT_PREFIX_INFO, ND_OPT_REDIRECT_HDR, ND_OPT_MTU, ND_OPT_NONCE, ND_OPT_ROUTE_INFO, ND_OPT_RDNSS, ND_OPT_DNSSL, ND_OPT_6CO
+
+# https://tools.ietf.org/html/rfc4861#section-4.6
+icmpv6_ndisc_option {
+	option_type	flags[icmpv6_ndisc_option_types, int8]
+	length	int8
```

Comment:
This does not look auto-formatted.
`make  generate` will take care of formatting, or `make format_sys` will do it separately.

Reply:
Perhaps this should be `bytesize[data, int8]`?

Reply:
or `bytesize[parent, int8]`?

Reply:
I think `bytesize8[data, int8]` is what we want. The length calculation for NDISC options is similar to that of the Hop-by-Hop options.